lib/taskana-core/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -109,7 +109,7 @@ void should_BeAbleToCreateNewTask_When_TaskCopy() throws Exception {
   @WithAccessId(user = "user-1-1")
   @Test
   void testCreateSimpleManualTask() throws Exception {
-
+    taskanaEngine.getConfiguration().setAddAdditionalUserInfo(true);
```

Comment:
Should we maybe create a separate test for this? This is separate functionality which doesn't quite fit the scope of this test I think.

 I'm also not sure what a "manual" Task is supposed to be in this scope ( old test Name)

Reply:
Creating a separate test sounds good :) Can I also change the test name to `should_createSimpleTask()` ?

Reply:
Yes i think this would be a better name. It's also good to change old test names according to our newer guidelines when adding/changing them, so this would be appreciated :)